main.cpp

```diff
@@ -1010,13 +1014,366 @@ Part 1e - Step 19: Request a review
 paste your code below
 */
 
+/*
+Thing 1) Castle
+5 properties:
+    1) number of gates (int)
+    2) name of king (std::string)
+    3) name of queen (std::string)
+    4) amount of taxes collected (float)
+    5) is at war (bool)
+3 things it can do:
+    1) collect tax from peasants.
+    2) open || close gates.
+    3) give orders to the kingdom.
+*/
+struct Castle
+{
+// 5 properties:
+    int numberOfGates = 8;
+    std::string nameOfKing = "Edward";
+    std::string nameOfQueen = "Magda";
+    float amountOfTaxesCollected = 349.77f;
+    bool isAtWar = false;
+
+    // NESTED UDT
+    struct FarmlandRevenue
+    {
+        int farmlandArces = 65;
+        int numberOfWorkers = 120;
+        float wheatSalePricePerKG = 1.8f;
+        std::string farmlandType = "Cereals and Grains";
+        bool isFarmlandInDraught = false;
```

Comment:
You do not need to include the class/struct name in any variable names.
`acres`, not `farmlandAcres`
`type`, not `farmlandType`
`isInDrought`, not `isFarmlandInDrought` 
- [ ] revise